lib/variable.js

```diff
@@ -79,7 +79,7 @@ module.exports = function environmentVariable (name, value) {
         );
       }
 
-      return Boolean(val);
+      return Boolean('false' === val ? 0 : val);
```

Comment:
Hi,

string insensitive is ok; i've seen in shell people using TRUE or FALSE for boolean 

as treating 0 as false i am against type coercion; if someone need boolean value than it shall be boolean not something which can be translated to boolean because of js coercion rules; also it makes the definition of what you expect, more easier to read

looking again over the code, i dont think we need that boilerplate to return a boolean value,
i think it will be easier:

`
return 'false' === val ? false : true;
`

there is allready a check so we can have only 2 values: string 'false' and string 'true'